File: Python/PGS/LEVEL1/소수 찾기/gunwoo.py
def prime(num):
    if num < 2:
        return False
    i = 2
    while i * i <= num: 
        if num % i == 0:
            return False
        i += 1
    return True

def solution(n):
    answer = 0
    i = 1 
    while i <= n:
        if prime(i): 
            answer += 1
        i += 1 
    return answer

# 시간초과 나는 반쪽짜리 정답

File: Python/PGS/LEVEL1/자연수 뒤집어 배열로 만들기/gunwoo.java
class Solution {
    public int[] solution(long n) {
        String strn = Long.toString(n);
        int index = 0;
        int[] answer = new int[strn.length()];
        for(int i = strn.length()-1; i >= 0; i--){
            answer[index] = strn.charAt(i) - '0'; //0의 아스키 값이 48 
            index++;
        }
        return answer;
    }
}